htdocs/comm/propal/class/propal.class.php

```diff
@@ -1693,6 +1693,9 @@ public function fetch($rowid, $ref = '', $ref_ext = '', $forceentity = 0)
 				$this->multicurrency_total_ht = $obj->multicurrency_total_ht;
 				$this->multicurrency_total_tva = $obj->multicurrency_total_tva;
 				$this->multicurrency_total_ttc = $obj->multicurrency_total_ttc;
+				
+				// Project
+				$this->fetch_project();
```

Comment:
I think it is still at wrong place: 
For standardization purpose, the fetch load only the object, the exteafields, and in some cases the lines of detail.
So can you move the fetch_project() into the caller of fetch() , just after the fetch() ?

Reply:
Hi, fetch_project moved just after the fetch as you requested.